test/parallel/test-tls-client-getephemeralkeyinfo.js

```diff
@@ -31,8 +32,8 @@ function test(size, type, name, next) {
   if (name) tls.DEFAULT_ECDH_CURVE = name;
 
   const options = {
-    key: key,
-    cert: cert,
+    key: fixturesKey,
+    cert: fixturesCert,
```

Comment:
if the `key` and `cert` variable names are not changed above, this can become just ...

```js
const options { key, cert, ciphers: cipher };
```